playbooks/roles/monit/templates/monitrc.j2

```diff
@@ -5,9 +5,11 @@ set idfile /var/lib/monit/id
 set statefile /var/lib/monit/state
 
 set httpd port 2812 and
-    use address localhost  # only accept connection from localhost
-    allow localhost        # allow localhost to connect to the server
+    use address localhost  # Bind monit to localhost
 
+    # Allow any ip to connect to monit. Binding monit to localhost
+    # and requiring basic auth (through nginx) should be sufficient to protect this interface.
+    allow 0.0.0.0/0.0.0.0 # allow any ip to connect to monit.
```

Comment:
After explicitly adding `{{ streisand_ipv4_address }}` to the allowed list. I realized that this may not always work. It then occurred to me that if we consider nginx + basic auth good enough to protect streisant vpn connection info; It's good enough protect the monit url path(especially since monit is being bound to localhost).  I don't see much of a win to explicitly whitelisting client Ip's, especially since that seems to actually cause problems.